### PR TITLE
Add XSLT for export internal format to DFG Viewer

### PR DESCRIPTION
**Pull request by Beacze**
Script contains all obligatory fields for DFG Viewer. Names of kitodo metadata elements are based on ruleset https://github.com/kitodo/kitodo-production/blob/master/Kitodo/rulesets/subhh.xml .

@solth and @sebastian-meyer : it would be nice if you could leave here your reviews :)

### CODE REVIEW

File: Kitodo/src/main/resources/xslt/kitodo2mods.xsl [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                    </mods:name>
+                </mods:mods>
+            </xsl:when>
+        </xsl:choose>
+    </xsl:template>
+
+    <!-- ### PlaceOfPublication ### -->
+    <xsl:template match="kitodo:metadata[@name='PlaceOfPublication']">
+        <mods:mods>
+            <mods:originInfo>
```
Comment (sebastian-meyer) [inline Kitodo/src/main/resources/xslt/kitodo2mods.xsl]: `<mods:originInfo>` should have a `@eventType="publication"` to distinguish it from e.g. reproduction metadata.
```suggestion
            <mods:originInfo eventType="publication">
```

File: Kitodo/src/main/resources/xslt/kitodo2mods.xsl [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                    </mods:placeTerm>
+                </mods:place>
+            </mods:originInfo>
+        </mods:mods>
+    </xsl:template>
+
+    <!-- ### PublisherName ### -->
+    <xsl:template match="kitodo:metadata[@name='PublisherName']">
+        <mods:mods>
+            <mods:originInfo>
```
Comment (sebastian-meyer) [inline Kitodo/src/main/resources/xslt/kitodo2mods.xsl]:
```suggestion
            <mods:originInfo eventType="publication">
```

File: Kitodo/src/main/resources/xslt/kitodo2mods.xsl [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                    <xsl:value-of select="normalize-space()"/>
+                </mods:publisher>
+            </mods:originInfo>
+        </mods:mods>
+    </xsl:template>
+
+    <!-- ### PublicationYear ### -->
+    <xsl:template match="kitodo:metadata[@name='PublicationYear']">
+        <mods:mods>
+            <mods:originInfo>
```
Comment (sebastian-meyer) [inline Kitodo/src/main/resources/xslt/kitodo2mods.xsl]:
```suggestion
            <mods:originInfo eventType="publication">
```

**Comment by Beacze** [comment]
@sebastian-meyer Thanks a lot for your comment! I have adjusted file according your advices. I have reordered selects according to documentation and grouped nodes in one template which just query certain values:

```
<mods:mods xmlns:mets="http://www.loc.gov/METS/"
           xmlns:mods="http://www.loc.gov/mods/v3"
           xmlns:kitodo="http://meta.kitodo.org/v1/">
   <mods:titleInfo>
      <mods:title>Second process</mods:title>
   </mods:titleInfo>
   <mods:originInfo eventType="publication">
      <mods:place>
         <mods:placeTerm type="text">Dresden</mods:placeTerm>
         <mods:publisher>SLUB</mods:publisher>
         <mods:dateIssued keyDate="yes" encoding="iso8601"/>
      </mods:place>
   </mods:originInfo>
   <mods:physicalDescription>
      <mods:extent/>
   </mods:physicalDescription>
   <mods:location>
      <mods:shelfLocator/>
      <mods:url/>
   </mods:location>
   <mods:recordInfo>
      <mods:recordIdentifier/>
   </mods:recordInfo>
   <mods:titleInfo>
      <mods:title/>
   </mods:titleInfo>
   <mods:originInfo eventType="publication">
      <mods:place>
         <mods:placeTerm type="text"/>
         <mods:publisher/>
         <mods:dateIssued keyDate="yes" encoding="iso8601"/>
      </mods:place>
   </mods:originInfo>
   <mods:physicalDescription>
      <mods:extent/>
   </mods:physicalDescription>
   <mods:location>
      <mods:shelfLocator/>
      <mods:url/>
   </mods:location>
   <mods:recordInfo>
      <mods:recordIdentifier/>
   </mods:recordInfo>
</mods:mods>
```
That's a result file which I got while testing my changes in online transformer.

**Comment by Beacze** [comment]
After adding conditional check if variable is set problem is gone. Only existing nodes are inserted.

**Pull request merged**